### PR TITLE
Admin: Show errors more clearly

### PR DESCRIPTION
**Pull request by tulimaki**
Admin: Activate first sidebar list item with errors
Admin: Make first language tab with errors work better

Refs SHOOP-1398, SHOOP-1396

### CODE REVIEW

File: shoop/admin/static_src/base/js/content-blocks.js [hunk 1]
```diff
@@ -21,4 +21,9 @@ $(function() {
             $(this).find(".block-title").addClass("mobile-error-indicator");
         }
     });
+    // Activate first sidebar-list-item with errors
+    var $firstSidebarListItemWithErrors = $("a.sidebar-list-item.errors").first();
+    if ($firstSidebarListItemWithErrors.length > 0) {
```
Comment (akx) [inline shoop/admin/static_src/base/js/content-blocks.js]: I don't think you actually even need to check for the length -- [the trigger would act on a collection of 0 element, i.e. not much at all](https://github.com/jquery/jquery/blob/master/src/event.js#L912-L916).

File: shoop/admin/static_src/base/js/content-blocks.js [hunk 2]
```diff
@@ -21,4 +21,9 @@ $(function() {
             $(this).find(".block-title").addClass("mobile-error-indicator");
         }
     });
+    // Activate first sidebar-list-item with errors
+    var $firstSidebarListItemWithErrors = $("a.sidebar-list-item.errors").first();
```
Comment (akx) [inline shoop/admin/static_src/base/js/content-blocks.js]: This could be const-bound :)

**Comment by tulimaki** [comment]
@akx also added one commit to make language-tabs errors work better.

File: shoop/admin/static_src/base/js/form-utils.js [hunk 3]
```diff
@@ -56,9 +56,10 @@ $(function() {
             }
             const tabHasErrors = ($tabPane.find(".has-error").length > 0);
             if(tabHasErrors) {
-                $tab.find("a").append($(" <div class=error-indicator><i class=\"fa fa-exclamation-circle\"></i></div>"));
+                const $tab_link = $tab.find("a");
```
Comment (akx) [inline shoop/admin/static_src/base/js/form-utils.js]: Use camelCase in JavaScript :)

File: shoop/admin/static_src/base/js/content-blocks.js [hunk 4]
```diff
@@ -21,4 +21,9 @@ $(function() {
             $(this).find(".block-title").addClass("mobile-error-indicator");
         }
     });
+    // Activate first sidebar-list-item with errors
+    const $firstSidebarListItemWithErrors = $("a.sidebar-list-item.errors").first();
+    if ($firstSidebarListItemWithErrors) {
```
Comment (akx) [inline shoop/admin/static_src/base/js/content-blocks.js]: No need for this `if` altogether, I meant (`$(selector)` never returns a falsy value).,

Just

```javascript
$("a.sidebar-list-item.errors").first().trigger("click");
```

should be plenty.

**Pull request merged**